docs/Tutorials/Create-Private-Network.md

```diff
@@ -56,6 +57,7 @@ Copy the following genesis definition to a file called `privateNetworkGenesis.js
 {
   "config": {
```

Comment:
Shouldn't the genesis contain a chain id parameter ? also if it has one, setting the network id in the cli would be useless as the default network id is the genesis chain id.

Reply:
Updated